p2p/module.go

```diff
@@ -98,11 +87,15 @@ func (m *p2pModule) Start() error {
 			go m.handleNetworkMessage(data)
 		}
 	}()
-	m.
-		GetBus().
+
+	m.GetBus().
 		GetTelemetryModule().
 		GetTimeSeriesAgent().
-		CounterIncrement(p2pTelemetry.P2P_NODE_STARTED_TIMESERIES_METRIC_NAME)
+		CounterRegister(
```

Comment:
You want to register the the metric at `Start` time or `Create` time and `Increment` it when the `Start` event has elapsed.

`CounterIncrement` should be the method to call in this section. For `CounterRegister`, I agree that `Start` is not the most suitable place for it, perhaps `Create` could be more suitable.

Reply:
Re registration - Start is more appropriate since we don't have access to the bus until then.

Re increment - thank you for catching my copy-pasta + lack of attention to detail error and the explanation.